### PR TITLE
feebacks review

### PR DESCRIPTION
**Pull request by regis-leray**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<meta property="og:image" content="http://www.trampoline.dev/logo.png">
 	<meta property="og:site_name" content="trampoline.dev scala montreal development software consulting functional programming stack safe type level zio">
 
-  <title>Trampoline</title>
+  <title>Trampoline - montreal software consulting team</title>
```
Comment (sergei-shabanau) [inline index.html:18]: Capital `M`ontreal

File: index.html [hunk 2]
```diff
@@ -77,7 +77,7 @@
     <div class="container">
       <div class="row">
         <div class="col-lg-12 text-center">
-          <h2 class="section-heading">Software tools</h2>
+          <h2 class="section-heading">Our expertise</h2>
           <h3 class="section-subheading">that solve real business problems</h3>
```
Comment (sergei-shabanau) [inline index.html:81]: It doesn't sound right anymore.

The original text was about "we build tools that solve ..."

File: index.html [hunk 3]
```diff
@@ -249,83 +151,55 @@ <h3 class="section-subheading text-muted"></h3>
...
+            <li>              
               <div class="timeline-panel">
                 <div class="timeline-heading">
-                  <h4>We know our craft</h4>
-                  <!-- <h4 class="subheading">Transition to Full Service</h4> -->
+                  <h4>We master our craft</h4>
                 </div>
                 <div class="timeline-body">
-                  <p class="text-muted">We don't just use those open source libraries, we build them.</p>
+                  <p class="text-muted">We are developers, we build better tools that improve solution on a user and business level.</p>
```
Comment (sergei-shabanau) [inline index.html:179]: The new text doesn't match with overall chapter. It became too general, and before it was specific. 
(the general section is above, and this one is intended to be special, with hard truth between lines as your friend advised us)

File: index.html [hunk 4]
```diff
@@ -338,7 +212,7 @@ <h4>Be Part
       <div class="row">
         <div class="col-lg-12 text-center">
           <h2 class="section-heading">World class team</h2>
-          <h3 class="section-subheading">Making impact by creating quality software, teaching others, speaking at conferences and maintaining open source projects.</h3>
+          <h3 class="section-subheading text-muted">Very different from others by creating quality software, teaching others, <br />speaking at conferences and maintaining open source projects.</h3>
```
Comment (sergei-shabanau) [inline index.html:215]: There is something wrong with English here. At least to my ear.

**Comment by soujiro32167** [comment]
Nice cleanup!
On another note, I have to say that maintaining this long term will be difficult...probably we will have to switch or get an FE at some point

File: index.html (hunk 3 above)
Comment (regis-leray) [reply index.html:179]: no problem we need to change the wording, not a big fan either 
```
We don't just use those open source libraries, we build them.
```
i prefer this one

```
we are not satisfied with the tools we have, that's why we make better tools.
```

File: index.html (hunk 2 above)
Comment (regis-leray) [reply index.html:81]: `Our expertise solve real business problems` ?

File: index.html (hunk 3 above)
Comment (regis-leray) [reply index.html]: we need to explain why we are building custom tools...

File: index.html (hunk 4 above)
Comment (regis-leray) [reply index.html:221]: 👍 i liked the suggestion from @alex 
`Very different from others, no consistency with others`

Comment (regis-leray) [reply index.html:221]:
```
We are builders, we don't just use open source libraries, we build them.
```

**Pull request merged**